src/screens/AboutScreen.js

```diff
@@ -17,51 +18,51 @@ import {
 import { getQuery } from '../queries';
 import { graphqlFetchPolicy } from '../helpers';
 
-export class AboutScreen extends React.PureComponent {
-  static contextType = NetworkContext;
+export const AboutScreen = ({ navigation }) => {
+  const { isConnected } = useContext(NetworkContext);
+  const fetchPolicy = graphqlFetchPolicy(isConnected);
+  const globalSettings = useContext(GlobalSettingsContext);
+  const { sections } = globalSettings;
+  const { headlineAbout = texts.homeTitles.about } = sections;
 
-  render() {
-    const { navigation } = this.props;
-    const isConnected = this.context.isConnected;
-    const fetchPolicy = graphqlFetchPolicy(isConnected);
-
-    return (
-      <SafeAreaViewFlex>
-        <Query
-          query={getQuery('publicJsonFile')}
-          variables={{ name: 'homeAbout' }}
-          fetchPolicy={fetchPolicy}
-        >
-          {({ data, loading }) => {
-            if (loading) {
-              return (
-                <LoadingContainer>
-                  <ActivityIndicator color={colors.accent} />
-                </LoadingContainer>
-              );
-            }
+  return (
+    <SafeAreaViewFlex>
+      <Query
+        query={getQuery('publicJsonFile')}
+        variables={{ name: 'homeAbout' }}
+        fetchPolicy={fetchPolicy}
+      >
+        {({ data, loading }) => {
+          if (loading) {
+            return (
+              <LoadingContainer>
+                <ActivityIndicator color={colors.accent} />
+              </LoadingContainer>
+            );
+          }
 
-            let publicJsonFileContent =
+          let publicJsonFileContent =
               data && data.publicJsonFile && JSON.parse(data.publicJsonFile.content);
 
-            if (!publicJsonFileContent || !publicJsonFileContent.length) return <VersionNumber />;
+          if (!publicJsonFileContent || !publicJsonFileContent.length) return <VersionNumber />;
 
-            return (
-              <ScrollView>
+          return (
+            <ScrollView>
+              {!!headlineAbout && (
                 <TitleContainer>
-                  <Title>{texts.homeTitles.about}</Title>
+                  <Title>{headlineAbout}</Title>
```

Comment:
instead of using the text in `texts.homeTitles.about` from texts.js file, we use the text in `headlineAbout`, which comes from the global settings object from the server. fetching does not happen here.